app/controllers/XiEoriController.scala

```diff
@@ -37,7 +37,7 @@ class XiEoriController @Inject() (xiEoriInformationRepository: XiEoriInformation
       case Some(xiEoriInformation) => Future.successful(Ok(Json.toJson(xiEoriInformation)))
       case None => retrieveXiEoriInformation(eori).map {
         case Some(xiEoriInformation) => Ok(Json.toJson(xiEoriInformation))
-        case None => xiEoriInformationRepository.set(eori, XiEoriInformation("", "", XiEoriAddressInformation("",None,"","", None)))
+        case None => xiEoriInformationRepository.set(eori, XiEoriInformation("", "", XiEoriAddressInformation("",Some(""),Some(""),None, None)))
```

Comment:
Could you please split the line (as below) over multiple lines as it has become very long (let me know if its not showing long on your machine)? Also better to add a val like below at the start of the method and use in place of ""

val emptyString = ""

          xiEoriInformationRepository.set(
          eori,
          XiEoriInformation(emptyString, emptyString, XiEoriAddressInformation(pbeAddressLine1 = emptyString)))

Reply:
Made changes to split into multiple lines and added an emptystring val.